### PR TITLE
Fix docker

### PR DESCRIPTION
**Pull request by Fa8Bit**

### CODE REVIEW

**Review by opourazar — APPROVED** [review]
Changes according to our findings on the working versions.

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-since it includes formatting
-and the dev container is build to integrate
-with the
-[`Live Server`](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
-extension directly.
+If you wish to run the minigame with hot reload in development mode, use the `docker-compose-dev.yaml` file.
+While using VSCode, make sure you have the [`Live Server`](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) extension installed, and the live server is active.
+You can start the live server by using the `Go Live` button in the bottom right corner of VSCode.
 
+You can the use the following commands to start up the dependencies in docker and the game itself with npm:
```
Comment (opourazar) [inline README.md:19]: Just a little typo here with 'the' in "You can the use... "

**Review by opourazar — COMMENTED** [review]
Your adjustments of the documentation reflect what we found out during our meeting with the former developer.

**Review by eisenmls — APPROVED** [review]
good

**Pull request merged**